File: doctor/doctdata/src/main/java/com/pecenio/doctms/model/Doctor.java
package com.pecenio.doctms.model;
import lombok.Data;
import java.util.Date;

@Data
public class Doctor{
	private int id;
	private Long userId;
	private String username;
	private String password;
	private String role;
	private String firstName;
	private String lastName;
	private String specialization;
	private String availabilityStatus;
	private Date lastUpdated;
	private Date created;
}

File: StaffApp/app/src/main/java/com/example/patientapp/data/model/Doctor.kt
package com.example.patientapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Doctor(
    val id: Long = 0,
    val userId: Long? = null,
    val username: String,
    val password: String? = null,
    val role: String = "DOCTOR",
    val firstName: String,
    val lastName: String,
    val specialization: String,
    val availabilityStatus: String = "AVAILABLE"
) : Parcelable
